packages/block-library/src/table/edit.js

```diff
@@ -291,14 +291,21 @@ export class TableEdit extends Component {
 
 		const { attributes, setAttributes } = this.props;
 		const { sectionName, rowIndex } = selectedCell;
+		const newRowIndex = rowIndex + delta;
 
-		this.setState( { selectedCell: null } );
 		setAttributes(
 			insertRow( attributes, {
 				sectionName,
-				rowIndex: rowIndex + delta,
+				rowIndex: newRowIndex,
 			} )
 		);
+		// this just allows to be able to add more rows, by enabling the corresponding
+		// buttons. It needs to be revised when programmatically focus to RichText is handled
+		this.createOnFocus( {
+			sectionName,
+			rowIndex: newRowIndex,
+			columnIndex: 0,
+		} );
```

Comment:
By definition, `createOnFocus` just returns a new function but doesn't do anything with it. So I'm fairly sure that nothing is happening here. :)

Reply:
Thanks Miguel. I couldn't understand why I left it that way... Maybe when I was trying to find how to focus on RichText..?

I changed it to just change the selected cell in state.